src/DotNet.Status.Web/appsettings.json

```
@@ -25,6 +25,12 @@
           "DefinitionPath": "\\dotnet\\helix-service",
           "Branches": [ "master", "production" ],
           "Assignee": "ChrisBoh"
+        },
+        {
+          "Project": "internal",
+          "DefinitionPath": "\\dotnet\\arcade-validation",
```

I think it needs to be this (the existing thing is wrong):
```suggestion
          "DefinitionPath": "\\dotnet\\arcade-validation\\dotnet-arcade-validation-official",
```

---

Also, I merge conflicted with you probably. :-)